requestidlecallback/callback-idle-periods.html

```diff
@@ -15,6 +15,7 @@
     var now = performance.now();
     var remaining = deadline.timeRemaining();
     var new_deadline = now + remaining;
+    while (now == performance.now());
```

Comment:
while (performance.now() <= new_deadline)

Reply:
That's not quite right. remaining can be a non-zero value.